### PR TITLE
Add in expect assertion cases

### PR DESCRIPTION
**Pull request by browniefed**
#13

### CODE REVIEW

**Comment by browniefed** [comment]
Had to skip over from the previous memberExpression structure since it didn't match how all the other conditions were done.

File: src/transformers/ava.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ import proxyquireTransformer from '../utils/proxyquire';
 
 const SPECIAL_THROWS_CASE = '(special throws case)';
 const SPECIAL_BOOL = '(special bool case)';
+const SPECIAL_ASSERTION_CASE = '(special assertion case)';
```
Comment (skovhus) [inline src/transformers/ava.js]: Small thing: but as everything is "assertions" can we call it `SPECIAL_PLAN_CASE`. That might be a bit easier to remember.

**Comment by skovhus** [comment]
One question might be if we should leave `t.plan` if there is no async looking code. In tape you usually have a lot of pointless `t.plan` if you are not using `t.end`... Any thoughts?

**Comment by browniefed** [comment]
Updated.
I would generally attempt to make less assumptions. In our case it won't matter, `expect.assertions` doesn't depend on things being async to work.

**Pull request merged**